circle.yml

```diff
@@ -479,7 +479,7 @@ commands:
               # internal PR
               if <<parameters.experimentalSessionAndOrigin>>; then
                 CYPRESS_RECORD_KEY=$MAIN_RECORD_KEY \
-                yarn cypress:run-experimentalSessionAndOrigin --record --parallel --group 5x-driver-<<parameters.browser>>-experimentalSessionAndOrigin --browser <<parameters.browser>>
+                DEBUG=cypress:webpac* yarn cypress:run-experimentalSessionAndOrigin --record --parallel --group 5x-driver-<<parameters.browser>>-experimentalSessionAndOrigin --browser <<parameters.browser>>
```

Comment:
still need this debug log?

Reply:
Nope - good catch! Removed in https://github.com/cypress-io/cypress/pull/23283/commits/dad0523765bc531abf2e30330133fc266e1fef3c